src/mon/HealthMonitor.h

```diff
@@ -45,9 +45,10 @@ class HealthMonitor : public QuorumService
   virtual bool service_dispatch(MonOpRequestRef op);
 
   virtual void start_epoch() {
+    epoch_t epoch = get_epoch();
     for (map<int,HealthService*>::iterator it = services.begin();
          it != services.end(); ++it) {
-      it->second->start(get_epoch());
```

Comment:
I agree with @tchaikov - I seriously doubt this will have any significant impact whatsoever in the monitor's performance. Much more so because this function is not heavily used. But I'm not opposed to merging this if the commit message is something more down the lines of making code readable rather than boasting an optimization.

Besides, unless we expect the state to change (i.e., get_epoch() to return a different value on each iteration, which we don't), it's just good practice to keep the get_epoch() out of the loop.